zrml/prediction-markets/src/lib.rs

```diff
@@ -357,7 +357,7 @@ mod pallet {
         #[pallet::weight(T::WeightInfo::cancel_pending_market())]
         pub fn cancel_pending_market(
             origin: OriginFor<T>,
-            market_id: MarketIdOf<T>,
+            #[pallet::compact] market_id: MarketIdOf<T>,
         ) -> DispatchResult {
             let sender = ensure_signed(origin)?;
 
```

Comment:
In `create_categorical_market` parameter `categories` is `u16` and can be compacted too.

Reply:
I didn't change `cancel_pending_market`, because that's on its way out the door. But good catch regarding `u16`, I didn't think about that.